__tests__/app.test.js

```diff
@@ -53,3 +53,41 @@ describe("/api endpoints", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET 200 AND return the correct article object with the correct properties and status 200", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
+      });
+  });
+
+  test('404 - responds with "Not Found" for non-existent id', () => {
+    return request(app)
+      .get("/api/articles/12334404")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe();
+        console.log("error 404 article not found")
+      });
+  });
+
+  test("400 - responds with error when id is NaN", () => {
```

Comment:
NaN means something specific in JS and although the description is technically correct it is a little misleading when used in this context. Would consider changing